chainspecs/devnetSpecRaw.json

```
@@ -2,9 +2,7 @@
   "name": "Creditcoin3 Dev",
   "id": "creditcoin3_dev",
   "chainType": "Live",
-  "bootNodes": [
-    "/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWFEX4ncCfVv2VqCZLTaPoGTuXQR4pY3M2aGV1eGSqpmUQ"
-  ],
+  "bootNodes": [],
```

Do you mind adding a check which makes sure that there are no boot nodes defined in any of the chain-specs ?

Option 1 - add it somewhere here: 
https://github.com/gluwa/creditcoin3/blob/dev/.github/workflows/ci.yml#L91

can be a simple grep/bash script or maybe slightly better parsing the json with `jq`.

Option 2  - we have a dedicated section for chainspec checks which executes one of the existing integration tests with a different config file: 
https://github.com/gluwa/creditcoin3/blob/dev/.github/workflows/ci.yml#L687


Either one will work but 1) should fail faster so is probably better.

---

Implemented 1